app/views/users/index.html.erb

```diff
@@ -0,0 +1,47 @@
+<%= stylesheet_link_tag 'style.css' %>
+<div class="user">
+  <div class="col-md-12">
+    <h4>Members</h4>
+    <button type="button" style="float: right; margin-top:10px;" class="btn btn-info" data-toggle="modal" data-target="#exampleModalCenter" <%= link_to 'Add Member', new_user_path %></button>
+    <div class="modal fade" id="exampleModalCenter" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true">
+      <div class="modal-dialog modal-dialog-centered" role="document">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h5 class="modal-title" id="exampleModalLongTitle">Modal title</h5>
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+              <span aria-hidden="true">&times;</span>
+            </button>
+          </div>
+        <div class="modal-body">
+          ...
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+          <button type="button" class="btn btn-primary">Save changes</button>
+        </div>
+      </div>
+    </div>
+  </div>
+  <table>
+    <tr>
+      <th>Name</th>
+      <th>Email</th>
+      <th>Role</th>
+      <th></th>
+      <th></th>
+    </tr>
+    <% @users.all.each do |user| %>
+      <% if user.organisation_id == current_user.organisation_id %>
+        <tr>
+          <td><%= user.first_name + " " + user.last_name %></td>
```

Comment:
use helper_method please instead on concatenating first_name and last_name here

Reply:
done